example/lib/page_remote_config.dart

```diff
@@ -34,7 +34,6 @@ class _RemoteConfigPageState extends State<RemoteConfigPage> {
 //===================================================
 // Contents
 //===================================================
-    void Contents() {}
```

Comment:
This is for quick access of contents. Below you will see comments with "return to [Constents]". When clicked it would take the user to here. Why remove? What is the alternative? If no alternative and we decide to remove this why not erase the comments?

Reply:
updated in next PR